Makefile

```
@@ -1,65 +1,13 @@
-all: test style-check
+all: style-check test
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)